advocacy_docs/partner_docs/VisionXGuide/06-CertificationEnvironment.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Certification Environment'
```

Comment:
For this section lets just keep

Certification Test Date
EDB Postgres Advanced Server
EDB Postgres Extended
SIB Visions VisionX
information.
The other lines from OS to Storage can be removed.